docs/ruleset_functions/expected_system_type_from_Table_G3_1_1.md

```diff
@@ -0,0 +1,117 @@
+		- the expected system is System 2: `system_type = "SYS-1"
+
+
+- check if the building area type is Public Assembly: `if building_area_type == "PUBLIC_ASSEMBLY":`
+	- check if the space area is less than 120,000 ft2: `if area_of_BAT < 120000:`
+		- set area_of_BAT_string to "< 120,000": `area_of_BAT_string = " < 120,000 ft2"`
+		- check if the climate zone is CZ_0_to_3a: `if is_CZ_0_to_3a():`
+			- the expected system is System 3: `system_type = "SYS-4"`
+		- otherwise, it's CZ_3b_3c_or_4_to_8: `else:`
+			- the expected system is System 4: `system_type = "SYS-3"
+	- otherwise, the space area is greater than 120,000 ft2: `else:`
+		- set area_of_BAT_string to ">= 120,000": `area_of_BAT_string = " >= 120,000 ft2"`
+		- check if the climate zone is CZ_0_to_3a: `if is_CZ_0_to_3a():`
+			- the expected system is System 12: `system_type = "SYS-13"`
+		- otherwise, it's CZ_3b_3c_or_4_to_8: `else:`
+			- the expected system is System 13: `system_type = "SYS-12"
+
+
+- check if the building area type is heated-only storage: `if building_area_type == "HEATED-ONLY_STORAGE":`
+	- check if the climate zone is CZ_0_to_3a: `if is_CZ_0_to_3a():`
+		- the expected system is System 9: `system_type = "SYS-10"`
+	- otherwise, it's CZ_3b_3c_or_4_to_8: `else:`
+		- the expected system is System 10: `system_type = "SYS-9"
+
+
+- check if the building area type is Retail: `if building_area_type == "RETAIL":`
+	- check if there are fewer than 3 floors in the building: `if number_of_floors < 3:`
+		- set number_of_floors_string to " 1 or 2 floors": `number_of_floors_string = " 1 or 2 floors"`
+		- check if the climate zone is CZ_0_to_3a: `if is_CZ_0_to_3a():`
+			- the expected system is System 3: `system_type = "SYS-4"`
+		- otherwise, it's CZ_3b_3c_or_4_to_8: `else:`
+			- the expected system is System 4: `system_type = "SYS-3"
+	- otherwise, WHAT HAPPENS WHEN RETAIL IS IN A BUILDING THAT IS MORE THAN 2 FLOORS?  FOR NOW, I'LL RECLASSIFY AS OTHER_NON_RESIDENTIAL: `else:`
+		- check if the space area is less than 120,000 ft2: `building_area_type = "OTHER_NON_RESIDENTIAL"
+
+
+- check if the building area type is Hospital: `if building_area_type == "HOSPITAL":`
+	- Hospital BAT doesn't use the climate zone to determine system type, so set climate_zone_category to "": `climate_zone_category = ""`
+	- check if the space area is more than 150,000 ft2 OR the building is more than 5 floors: `if((area_of_BAT > 150000) || (number_of_floors > 5)):`
+		- set area_of_BAT_string to " > 150,000 ft2 or > 5 floors": `area_of_BAT_string = " > 150,000 ft2 or > 5 floors"`
+		- the expected system is System 7: `system_type = "SYS-7"`
+	- otherwise: `else:`
+		- set area_of_BAT_string to " All Other": `area_of_BAT_string = " All Other"`
+		- the expected system is System 5: `system_type = "SYS-5"`
+
+
+- check if the building area type is other non-residential: `if building_area_type == "OTHER_NON_RESIDENTIAL":`
+	- check if the space area is less than 25,000: `if area_of_BAT < 25000:`
```

Comment:
to check space area, need to cycle through all zones in the building segment and calculate the conditioned area. Could use this function as an example to create this new function (if required)
https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/ruleset_functions/get_hvac_zone_list_w_area.md

Reply:
building_area_type is passed into the function. It is calculated using the function [get_HVAC_building_area_types_and_zones](https://github.com/pnnl/ruleset-checking-tool/blob/RDS/KJW/function_get_HVAC_building_area_types_and_zones/docs/ruleset_functions/get_HVAC_building_area_types_and_zones.md)